cmd/deploy.go

```diff
@@ -27,6 +28,8 @@ type deployCmd struct {
 	authArgs
 
 	apimodelPath      string
+	dnsPrefix         string
+	autoSuffix        bool
```

Comment:
Not right now, I think integration test might be more important. In terms of what I'd unit test in deploy.go, the issue I filed about changing validation would let us formalize how the default setting happens in a much more testable fashion.